### PR TITLE
Create release note for FT 1.1 and tidy up the spec

### PR DESCRIPTION
**Pull request by Emily-Jiang**
add release note for FT 1.1
tidy up the spec to make it readable

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
@Azquelt please review as well

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 1]
```diff
@@ -111,12 +104,26 @@ If an annotation is not present, the configured properties are ignored.
 For instance, the property `com.acme.ClassA/methodB/Retry/maxRetries` will be ignored if `@Retry` annotation is not specified on the `methodB` of `com.acme.ClassA`.
 Similarly, the property `com.acme.ClassA/Retry/maxRetries` will be ignored if `@Retry` annotation is not specified on the class `com.acme.ClassA` as a class-level annotation.
 
-=== Disabled Fault Tolerance policies with configuration
+=== Disable a group of Fault Tolerance annotations on the global level
+In some service mash platform, e.g. Istio, has its own Fault Tolerance policy.
```
Comment (OndrejM) [inline spec/src/main/asciidoc/configuration.asciidoc]: This sentence doesn't make sense. It should probably be "**Some** service mesh **platforms**, e.g. Istio, **have their own** Fault Tolerance policy"

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 2]
```diff
@@ -111,12 +104,26 @@ If an annotation is not present, the configured properties are ignored.
...
+The operation team might want to use the platform Fault Tolerance.
+In order to fulfil the requirement, MicroProfile Fault Tolerance provides a capability to have its resilient functionalities except `fallback` disabled.
```
Comment (OndrejM) [inline spec/src/main/asciidoc/configuration.asciidoc]: "functionalities disabled except `fallback`" makes much more sense to me grammatically.

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 3]
```diff
@@ -111,12 +104,26 @@ If an annotation is not present, the configured properties are ignored.
...
+The reason `fallback` is special as the `fallback` business logic can only be defined by microservices not by any other platforms.
```
Comment (OndrejM) [inline spec/src/main/asciidoc/configuration.asciidoc]: "is special **is that the** `fallback`" ... "by microservices **and not by** any other" - grammar fixes

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 4]
```diff
@@ -111,12 +104,26 @@ If an annotation is not present, the configured properties are ignored.
...
+
+Set the config property of `MP_Fault_Tolerance_NonFallback_Enabled` with the value of `false` means the Fault Tolerance is disabled, except `@Fallback`.
```
Comment (OndrejM) [inline spec/src/main/asciidoc/configuration.asciidoc]: "**Setting** the config property" - grammar

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 5]
```diff
@@ -160,9 +167,11 @@ Policy will be disabled everywhere ignoring existing policy annotations on metho
 
 If the above configurations patterns are used with a value other than `true` or `false` (i.e. `<classname>/<methodname>/<annotation>/enabled=whatever`) non-portable behaviour results.
 
-When the above property is used together with the property `MP_Fault_Tolerance_NonFallback_Enabled`, the propoerty `MP_Fault_Tolerance_NonFallback_Enabled` has the lowest priority. 
-`MP_Fault_Tolerance_NonFallback_Enabled=true`
-`Bulkhead/enabled=true`
-All other annotations except `Fallback` and `Bulkhead` will not be enabled.
+When the above property is used together with the property `MP_Fault_Tolerance_NonFallback_Enabled`, the propoerty `MP_Fault_Tolerance_NonFallback_Enabled` has the lowest priority. e.g.
```
Comment (OndrejM) [inline spec/src/main/asciidoc/configuration.asciidoc]: "the property" - typo

**Pull request merged**